genesis/utils/mjcf.py

```diff
@@ -472,6 +472,8 @@ def parse_equality(mj, i_e, scale, ordered_links_idx):
         e_info["eq_obj2id"] = -1 if mj.eq_obj2id[i_e] == 0 else ordered_links_idx.index(mj.eq_obj2id[i_e] - 1)
     elif mj.eq_type[i_e] == mujoco.mjtEq.mjEQ_WELD:
```

Comment:
just do `mj.eq_type[i_e] in (mujoco.mjtEq.mjEQ_CONNECT, mujoco.mjtEq.mjEQ_WELD)`

Reply:
still need to set `e_info["type"]` separately